kafka/integration_test.go

```diff
@@ -2519,6 +2519,57 @@ func (its *IntegrationTestSuite) TestProducerConsumerHeaders() {
 
 }
 
+// TestImmediateFlush tests that producer.Flush ignores
+// "queue.buffering.max.ms". (Issue #1013).
+func (its *IntegrationTestSuite) TestImmediateFlush() {
```

Comment:
This test doesn't seem to depend on the `IntegrationTestSuite` fixtures since it's using the mock cluster.

So I suggest to make a normal test, not a method of `IntegrationTestSuite`. This will make it run by default, without needing to setup a `testconf.json`.

Reply:
Good thought, I made a different file for mock-cluster based tests.